packages/tools/bcm2835-bootloader/scripts/update.sh

```diff
@@ -50,6 +50,9 @@
 #    sed -e "s,gpu_mem=100,gpu_mem=128,g" -i $BOOT_ROOT/config.txt
 #    sed -e "s,gpu_mem_256=100,# gpu_mem_256=128,g" -i $BOOT_ROOT/config.txt
 #    sed -e "s,gpu_mem_512=128,# gpu_mem_512=128,g" -i $BOOT_ROOT/config.txt
+  # Add distribution specific extra configuration file for the bootloader
+  elif [ -f $SYSTEM_ROOT/usr/share/bootloader/distroconfig.txt ]; then
```

Comment:
Shouldn't this be an `if` (rather than `elif) so that the `distroconfig.txt` is **always** copied (if it exists)?

As it is, should the user modify their `config.txt` in such a way that either of the previous conditions are met, the `distroconfig.txt` won't be copied.